.github/workflows/test.yml

```diff
@@ -44,6 +34,28 @@ jobs:
       - name: Run tests
         run: pytest -n 4 tests --disable-warnings --generate-plots
 
+  test-mac-win:
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.12"]
+        os: [macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup xtgeo
+        uses: "./.github/actions/setup_xtgeo"
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup testdata
+        uses: "./.github/actions/setup_testdata"
+
+      - name: Run tests
+        run: pytest -n 4 tests --disable-warnings
```

Comment:
Why not -n auto?

Reply:
auto defaults to the number of physical cores and GitHub limits them on action runners to 2. Specifying in addition will have xdist run additional workers per core and 4 in the past seemed a sweet spot